src-tool/Pact/Analyze/Check.hs

```diff
@@ -452,23 +451,26 @@ verifyFunctionInvariants tf (CheckEnv tables _consts _pDefs moduleData caps gov
     catchingExceptions act = act `E.catch` \(e :: SBV.SBVException) ->
       pure $ Left $ CheckFailure funInfo $ SmtFailure $ UnexpectedFailure e
 
+    modName = moduleDefName $ _mdModule moduleData
+
+    debugFile = if debug then Just $ ".pact-verify-"<> asString' modName else Nothing
```

Comment:
As a note, `asString` uses the namespace prefixed module name.

Reply:
@jmcardon , I need to write to several temporary files for this feature. Is there any policy/library/functions in-place within pact?

Reply:
I don't believe so, but I think this should be doable with `System.FilePath`+ io bracket.